Python/ceval.c

```diff
@@ -988,6 +990,43 @@ _PyObjectArray_Free(PyObject **array, PyObject **scratch)
     }
 }
 
+#if _Py_TIER2
+// 0 for success, -1  for error.
+static int
+bail_tracing_and_jit(PyThreadState *tstate, _PyInterpreterFrame *frame)
```

Comment:
"Bail" suggests we are abandoning the trace. But we aren't, maybe `stop_tracing_and_jit`?